### PR TITLE
release note for 0.9.1

### PR DESCRIPTION
**Pull request by y-f-u**
v0.9.1-alpha release note

### CODE REVIEW

File: docs/release_notes/v0.9.1-alpha.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQlite data store.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
- **Adds** SQLite data store (`sqlite`).
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- **Adds** support for date, varchar, bpchar, primitive list type in Postgres.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
- **Adds** support for date, varchar, bpchar, and primitive list types for the PostgreSQL data connector and data store.
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- **Adds** `spice pods`, `spice status`, `spice datasets`, and `spice models` CLI commands.
+- **Adds** `v1/spicepods` API.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
- **Adds** `GET /v1/spicepods` API for listing loaded Spicepods.
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- **Adds** `spiced` docker build and release.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
- **Adds** `spiced` Docker CI build and release.
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- **Adds** e2e tests for release installation and local acceleration.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
- **Adds** E2E tests for release installation and local acceleration.
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- **Fixes** `spice sql` REPL panics when query result is too large.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]: @y-f-u Please add the Issue # reference for bug fixes.

Comment (mitchdevenport) [reply docs/release_notes/v0.9.1-alpha.md]: Also, maybe just say `bug fixes for spice sql` and then list them

File: docs/release_notes/v0.9.1-alpha.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+- **Fixes** `--access-secret`
```
Comment (mitchdevenport) [inline docs/release_notes/v0.9.1-alpha.md]: What does it fix?

File: docs/release_notes/v0.9.1-alpha.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+The v0.9.1 release adds `flightsql` data connector and `sqlite` data store. Improved the usability by adding more `spice` CLI commands.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added. 
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+## Highlights in v0.9.1-alpha
+
+**FlightSQL data connector**: [Arrow Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) can now be used as a connector for federated SQL query.
+
+**SQlite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
**SQLite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
```

File: docs/release_notes/v0.9.1-alpha.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQLite data store (`sqlite`).
+- **Adds** support for date, varchar, bpchar, and primitive list types for the PostgreSQL data connector and data store.
+- **Adds** `spice pods`, `spice status`, `spice datasets`, and `spice models` CLI commands.
+- **Adds** `GET /v1/spicepods` API for listing loaded Spicepods.
+- **Adds** `spiced` Docker CI build and release.
+- **Adds** E2E tests for release installation and local acceleration.
```
Comment (sgrebnov) [inline docs/release_notes/v0.9.1-alpha.md:27]: Please add
- **Adds** E2E tests and instructions to run basic [TPC-H benchmark](https://www.tpc.org/tpch/) tests

File: docs/release_notes/v0.9.1-alpha.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQLite data store (`sqlite`).
```
Comment (mitchdevenport) [inline docs/release_notes/v0.9.1-alpha.md]: Might want to mention support for in-memory and file based since not all data stores support both

**Review by phillipleblanc — DISMISSED** [review]

File: docs/release_notes/v0.9.1-alpha.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
+The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added.
```
Comment (lukekim) [inline docs/release_notes/v0.9.1-alpha.md]:
```suggestion
The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status`) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added. 
```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**